packages/cli/src/plugins/resource/plugin-node-modules.js

```diff
@@ -76,13 +77,25 @@ const walkModule = (module, dependency) => {
       if (sourceValue !== '' && sourceValue.indexOf('.') !== 0 && sourceValue.indexOf('http') !== 0) {
         updateImportMap(sourceValue, `/node_modules/${sourceValue}`);
       }
+
+      if (sourceValue.indexOf('.') === 0 && fs.existsSync(path.join(process.cwd(), 'node_modules', dependency, sourceValue))) {
+        const moduleContents = fs.readFileSync(path.join(process.cwd(), 'node_modules', dependency, sourceValue));
+
+        walkModule(moduleContents, dependency);
+      }
     },
     ExportAllDeclaration(node) {
       const sourceValue = node && node.source ? node.source.value : '';
 
       if (sourceValue !== '' && sourceValue.indexOf('.') !== 0 && sourceValue.indexOf('http') !== 0) {
         updateImportMap(sourceValue, `/node_modules/${sourceValue}`);
       }
+
+      if (sourceValue.indexOf('.') === 0 && fs.existsSync(path.join(process.cwd(), 'node_modules', dependency, sourceValue))) {
+        const moduleContents = fs.readFileSync(path.join(process.cwd(), 'node_modules', dependency, sourceValue));
```

Comment:
In the context of #557 , it seems I did indeed overlook other usages of hardcoded _node_modules_ usages like here, which kind of voids the safety intended by landing #674 .  🤦 

In a trivial example like on our home page, `npx` should work, but if a _package.json_ is introduced, the logic here could have potential to break / behave unexpectedly so just wanted to call this out.